RELEASENOTES-1.4.docu

```diff
@@ -288,4 +288,14 @@
       <tt>cancel_after()</tt> method, provided as part of the <tt>object</tt>
       template <bug number="SIMICS-17930"/>.
   </add-note></build-id>
+  <build-id value="next"><add-note> When declaring an object array, any
+      dimension size specification may now be omitted if already defined a
```

Comment:
grammatical errors here:
"if already defined a different declaration of the same object array" should probably be along the lines of "if another declaration of the same object array which defines the size exists"

Reply:
oops